### PR TITLE
Bug/configuration is not published

### LINKED ISSUE #30: Configuration values are not correctly retrieved
Configuration values are not retrieved correctly when asked via configuration topics. The problem is not detected in the tests, as a test validating the publishing format is also missing.

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #30

### CODE REVIEW

File: test/unit/get-configuration_test.js [hunk 1]
```diff
@@ -93,11 +93,11 @@ describe('Get configuration from the devices', function() {
                 .reply(200,
                 utils.readExampleFile('./test/contextResponses/getConfigurationSuccess.json'));
 
-            configurationReceived = false;
+            mqttClient.subscribe('/1234/MQTT_2/configuration/values', null, function(error, values, pruebas) {
```
Comment (gtorodelvalle) [inline test/unit/get-configuration_test.js]: `pruebas`??? In fact, it is really needed to pass this empty function??? :)

**Comment by XavierVal** [comment]
LGTM

File: test/unit/get-configuration_test.js (hunk 1 above)
Comment (dmoranj) [inline test/unit/get-configuration_test.js]: Fixed in 76b78ac

**Comment by gtorodelvalle** [comment]
LGTM and merging!!! ;) Thanks @dmoranj :)

**Pull request merged**